src/test/java/com/microsoft/sqlserver/jdbc/unit/lobs/LobsTest.java

```diff
@@ -9,12 +9,16 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import java.io.BufferedInputStream;
+import java.io.BufferedReader;
```

Comment:
I think you should add a case where you create a stream out of a ResultSet, close the resultset, then try to read from the stream. The tests below test that existing functionality is still supported, but doesn't verify the case that was intended to be fixed in this PR.

Reply:
testContinuousReading() tests the case that was intended to be fixed.